Standards/CCSMB-5.md

```diff
@@ -0,0 +1,22 @@
+# CCSMB-5: Radio Transmissions
+```md
+*Author: Emma Beurskens <@emmaknijn>*
+*Version: v1.0.0*
+*Last updated: YYYY-MM-DD*
+```
+
+## Design
+A client will only listen to packets while the audio is being transmitted from a server
+The server will communicate to the client over modem
+
+## Packet
+A packet consists of the following fields in a table:
+- buffer: Signed 8-bit PCM @ 48kHz
```

Comment:
Might want to elaborate on this later on.

Reply:
znepb suggested that that might be a standard as well